mlir/lib/Dialect/Vector/Transforms/VectorEmulateNarrowType.cpp

```diff
@@ -879,8 +881,7 @@ static Value rewriteI4ToI8SignedExt(PatternRewriter &rewriter, Location loc,
     interleaveMaskValues.push_back(i + (vecDimSize / 2));
   }
```

Comment:
Can the `interleaveMaskValues` (and the loop generate it) be removed now? It looks unused.

Also `// 3. Interleave low and high i8 elements using a shuffle.` is now outdated :)